runtime/runtimetools/perfscripts/perf-hottest

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
```

Comment:
This script should have a copyright header.

Reply:
Added